plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/flex/AppEngineFlexProjectWizardTest.java

```diff
@@ -21,10 +21,10 @@
 
 public class AppEngineFlexProjectWizardTest {
 
-  private AppEngineFlexProjectWizard wizard = new AppEngineFlexProjectWizard();
+  private final AppEngineFlexProjectWizard wizard = new AppEngineFlexProjectWizard();
 
   @Test
-  public void testWindowTitle() {
+  public void testTitleSet() {
```

Comment:
`testWindowTitle` seems more to the point, since it's testing the actual value of the window title, not just that it's set.